### PR TITLE
[DOCS] Fix local build directory in README.asciidoc

### PR DESCRIPTION
**Pull request by peterlamar**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by peterlamar** [comment]
Clarifying Building from source instructions to be more accurate. There does not appear to be a 'build/distribution' folder after running ./gradlew assemble

**Review by jrodewig — CHANGES_REQUESTED** [review]
Thanks for raising this @peterlamar. However, I think the more relevant output directory for most users is `distributions/archives`. 

I left a suggestion which corrects the directory and adds some os-specific commands too.
I'm also going to request a review from one of our engineers.

**Comment by jrodewig** [comment]
@pugnascotia Do you mind taking a look at the suggestion and confirm I'm not overlooking anything. Thanks!

File: README.asciidoc [hunk 1]
```diff
@@ -194,7 +194,7 @@ Elasticsearch uses https://gradle.org[Gradle] for its build system.
 
 In order to create a distribution, simply run the `./gradlew assemble` command in the cloned directory.
 
-The distribution for each project will be created under the `build/distributions` directory in that project.
+The distribution will be created under the `distribution/build/outputs` directory in the cloned directory root.
```
[the remark this answers is not in the document]
Comment (peterlamar) [reply README.asciidoc]: Agreed, much better. Incorporated your changes

File: README.asciidoc [hunk 2]
```diff
@@ -194,7 +194,17 @@ Elasticsearch uses https://gradle.org[Gradle] for its build system.
 
 In order to create a distribution, simply run the `./gradlew assemble` command in the cloned directory.
 
-The distribution for each project will be created under the `build/distributions` directory in that project.
+To build distributions from your local Elasticsearch repo, run:
```
Comment (mark-vieira) [inline README.asciidoc:197]: FWIW, you can now run `./gradlew localDistro` which will build the appropriate distribution for the current platform. The task also prints the output location for you so you don't have to guess about searching for it. We should probably mention it here as it's the preferred way to locally build Elasticsearch and get up and running quickly.

Comment (peterlamar) [reply README.asciidoc]: Updated, even better! Now its really clear how to get it up and running

**Comment by pugnascotia** [comment]
Nothing to add from me.

@elasticmachine ok to test

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks for the PR @peterlamar.

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Comment by jrodewig** [comment]
@elastimachine run elasticsearch-ci/default-distro

**Comment by jrodewig** [comment]
@elastimachine run elasticsearch-ci/bwc

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Pull request merged**